sympy/geometry/util.py

```diff
@@ -68,6 +68,8 @@ def are_coplanar(*e):
     from sympy.geometry.line import LinearEntity3D
     from sympy.geometry.point import Point3D
     from sympy.geometry.plane import Plane
+    from sympy.geometry.entity import GeometryEntity
```

Comment:
Why is this imported? It doesn't look like anything else is changed in this file.

Reply:
it is being used in line 107 in 'are_coplanar' functionn and was not imported. i found this now that it is imported in most of the function separately so done for this function too.